### ISSUE TITLE
Add protected B toggle in shiny/reports

### ISSUE THREAD

**Issue by stoyelq** [dfo-mar-odis/shinySpatialApp#61]
Add a checkbox to null all of the protected B data in the reports.

**Comment by gomezcatalina** [comment]
Yes! Is this something you did for the report you rendered yesterday? or a nice to have feature to-be-developed?

**Comment by stoyelq** [comment]
Yesterday, I just did it manually, it's still just something that would be nice to add to the report.
